### PR TITLE
UKIS schema changes

### PR DESCRIPTION
**Pull request by dwyeradam**
### Content Changes:
Wording changes and
Two additional blocks 9-9 and 9-10

### Routing Changes:
If Question 6.14 (`question2483`) is shown then Section 7 (`section611`) and Section 8 (`section612`) should not be shown. Neither should the majority of blocks in Section 9 (`section613`). I have therefore split section 9 into 3 groups, the introduction (always shown), the group containing questions that can be skipped and a final group of blocks that are always shown. 
Skip conditions are common across the 3 skipped groups (Section 7, 8 and 9b) and is the reverse of the skip condition on question 6.14.

### CODE REVIEW

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Undocumented but can you remove this additional space after `</em>`
https://github.com/ONSdigital/eq-survey-runner/blob/5b226ce8f16fc4b024f94a765a8970478d885c93/data/en/ukis_0001.json#L2308

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
While you're here?

Missing full stop on multiple answer descriptions:
`Enter the full value (e.g. 56,234.33) or a value to the nearest £thousand (e.g. 56,000) Do not enter ‘56’ for £56,000`

Should be:
`Enter the full value (e.g. 56,234.33) or a value to the nearest £thousand (e.g. 56,000). Do not enter ‘56’ for £56,000`

File: data/en/ukis_0001.json [hunk 1]
```diff
@@ -3539,6 +3560,100 @@
...
+                                    ]
+                                },
+                                {
+                                    "id": "answer9-9-exclusive",
+                                    "mandatory": false,
+                                    "type": "Checkbox",
+                                    "label": "",
+                                    "description": "",
+                                    "options": [{
+                                        "label": "No this business did not co-operate on innovation activities with other businesses outside your enterprise group",
```
Comment (MebinAbraham) [inline data/en/ukis_0001.json]: The slide was updated today. 🤷‍♂️ 
So this need changing to `the enterprise group` from `your enterprise group`

Comment (dwyeradam) [reply data/en/ukis_0001.json]: Does this also need doing for `answer3282-exclusive` in `block2532`? I've assumed so but will confirm with Donna. - Slide 82 Question 9.5

```
If {{ metadata.trad_as_or_ru_name }} co-operated on any innovation activities with <em>competitors or other businesses in your industry</em>, what was their location?

No, this business did not co-operate on innovation activities with competitors or other businesses in your industry
```

Comment (dwyeradam) [reply data/en/ukis_0001.json]: Wording of `block2534` needs confirming as well. - Slide 84 Question 9.7

```
If {{ metadata.trad_as_or_ru_name }} co-operated on any innovation activities with <em>universities or other higher education institutions</em>, what was their location?

No, this business did not co-operate on innovation activities with other businesses with the enterprise group
```

I've assumed it's wrong and therefore changed it to:
`No, this business did not co-operate on innovation activities with universities or other higher education institutions`

Comment (dwyeradam) [reply data/en/ukis_0001.json]: And `block2535` - Slide 85 Question 9.8

```
If {{ metadata.trad_as_or_ru_name }} co-operated on any innovation activities with <em>government or public research institutes</em>, what was their location?

No, this business did not co-operate on innovation activities with other businesses with the enterprise group
```

I've assumed it's wrong and therefore changed it to:
`No, this business did not co-operate on innovation activities with government or public research institutes`

**Comment by MebinAbraham** [comment]
@dwyeradam Worth sending a quick launch to someone from the business at this stage?

File: data/en/ukis_0001.json (hunk 1 above)
Comment (dwyeradam) [reply data/en/ukis_0001.json]: Donna has confirmed my assumptions above are correct.

**Review by MebinAbraham — COMMENTED** [review]
Tested the main bit of routing and it seems to work as expected.

Although as a side effect to splitting section 9 into multiple groups, the summary will show multiple collapsible sections. We need to either skip the individual blocks by repeating the conditions or ensure this is fine?

screenshot 2018-11-19 at 11 02 27

**Comment by dwyeradam** [comment]
To fix the summary i'm going to have to change from skip conditions to routing instead.

**Review by MebinAbraham — APPROVED** [review]
It will do 😁

**Review by bitdivision — APPROVED** [review]
Have checked all the points in trello, but haven't checked every possibility for routing on 6.14. The change itself looks good though.

**Pull request merged**